### PR TITLE
🌞 NSRDB - 8543 - Updating GOES 2023 Data Ingestion (V4.0.0) Documentation

### PR DESCRIPTION
**Pull request by msherman32**
This PR updates the attributes documentation for the following NSRDB datasets:

- Nsrdb Goes Aggregated V4.0.0
- Nsrdb Goes Conus V4.0.0
- Nsrdb Goes Full Disc V4.0.0
- Nsrdb Goes Tmy V4.0.0

resolves https://github.nrel.gov/vadr/vadr/issues/8543

### CODE REVIEW

**Review by PjEdwards — CHANGES_REQUESTED** [review]
Getting there. Just a quick change, probably a copy-paste thing.

File: source/docs/solar/nsrdb/nsrdb-GOES-aggregated-v4-0-0-download.html.md.erb [hunk 1]
```diff
@@ -57,7 +57,7 @@ _NOTE: when using POST to submit a request the api_key must still be included as
       <td class="doc-parameter-value">
         <div class="doc-parameter-value-field"><strong>Type:</strong> comma delimited string array</div>
         <div class="doc-parameter-value-field"><strong>Default:</strong> Returns ALL</div>
-        <div class="doc-parameter-value-field"><strong>Options:</strong> <em>dew_point, dhi, dni, ghi, surface_albedo, surface_pressure, air_temperature, wind_direction, wind_speed.</em></div>
+        <div class="doc-parameter-value-field"><strong>Options:</strong> <em>air_temperature, alpha, aod, asymmetry, clearsky_dhi, clearsky_dni, clearsky_ghi, cloud_fill_flag, cloud_type, dew_point, dhi, dni, fill_flag, ghi, ozone, relative_humidity, solar_zenith_angle, ssa, surface_albedo, surface_pressure, tmy_year, total_precipitable_water, wind_direction, wind_speed.</em></div>
```
[the remark this answers is not in the document]
Comment (msherman32) [reply source/docs/solar/nsrdb/nsrdb-GOES-aggregated-v4-0-0-download.html.md.erb]: [Resolved](https://github.com/NREL/developer.nrel.gov/pull/367/commits/b80e31b7774d839aa8acb1f2144883019a8ea70b)

File: source/docs/solar/nsrdb/nsrdb-GOES-conus-v4-0-0-download.html.md.erb [hunk 2]
```diff
@@ -57,7 +57,7 @@ _NOTE: when using POST to submit a request the api_key must still be included as
       <td class="doc-parameter-value">
         <div class="doc-parameter-value-field"><strong>Type:</strong> comma delimited string array</div>
         <div class="doc-parameter-value-field"><strong>Default:</strong> Returns ALL</div>
-        <div class="doc-parameter-value-field"><strong>Options:</strong> <em>dew_point, dhi, dni, ghi, surface_albedo, surface_pressure, air_temperature, wind_direction, wind_speed.</em></div>
+        <div class="doc-parameter-value-field"><strong>Options:</strong> <em>air_temperature, alpha, aod, asymmetry, clearsky_dhi, clearsky_dni, clearsky_ghi, cloud_fill_flag, cloud_type, dew_point, dhi, dni, fill_flag, ghi, ozone, relative_humidity, solar_zenith_angle, ssa, surface_albedo, surface_pressure, tmy_year, total_precipitable_water, wind_direction, wind_speed.</em></div>
```
[the remark this answers is not in the document]
Comment (msherman32) [reply source/docs/solar/nsrdb/nsrdb-GOES-conus-v4-0-0-download.html.md.erb]: [Resolved](https://github.com/NREL/developer.nrel.gov/pull/367/commits/b80e31b7774d839aa8acb1f2144883019a8ea70b)

File: source/docs/solar/nsrdb/nsrdb-GOES-full-disc-v4-0-0-download.html.md.erb [hunk 3]
```diff
@@ -57,7 +57,7 @@ _NOTE: when using POST to submit a request the api_key must still be included as
       <td class="doc-parameter-value">
         <div class="doc-parameter-value-field"><strong>Type:</strong> comma delimited string array</div>
         <div class="doc-parameter-value-field"><strong>Default:</strong> Returns ALL</div>
-        <div class="doc-parameter-value-field"><strong>Options:</strong> <em>dew_point, dhi, dni, ghi, surface_albedo, surface_pressure, air_temperature, wind_direction, wind_speed.</em></div>
+        <div class="doc-parameter-value-field"><strong>Options:</strong> <em>air_temperature, alpha, aod, asymmetry, clearsky_dhi, clearsky_dni, clearsky_ghi, cloud_fill_flag, cloud_type, dew_point, dhi, dni, fill_flag, ghi, ozone, relative_humidity, solar_zenith_angle, ssa, surface_albedo, surface_pressure, tmy_year, total_precipitable_water, wind_direction, wind_speed.</em></div>
```
[the remark this answers is not in the document]
Comment (msherman32) [reply source/docs/solar/nsrdb/nsrdb-GOES-full-disc-v4-0-0-download.html.md.erb]: [Resolved](https://github.com/NREL/developer.nrel.gov/pull/367/commits/b80e31b7774d839aa8acb1f2144883019a8ea70b)

**Review by PjEdwards — APPROVED** [review]
Perfect, thanks!

**Pull request merged**